### PR TITLE
Gooncode

### PR DESCRIPTION
**Pull request by Pyko1**
Adds a disk full of Gooncode to RD.
Has 4 levels of data research saved in it.
Also a traitor objective.

### CODE REVIEW

**Comment by Spacedong** [comment]
ok i kekked

**Comment by Nienhaus** [comment]
:+1: x :100:

**Comment by Carbonhell** [comment]
should be atleast level 5 or 6

**Comment by Memendia** [comment]
This is great

File: code/game/gamemodes/objective_items.dm [hunk 1]
```diff
@@ -59,6 +59,12 @@
 	difficulty = 5
 	excludefromjob = list("Captain")
 
+/datum/objective_item/steal/gooncodedisk
+	name = "the Gooncode disk"
```
Comment (Carbonhell) [inline code/game/gamemodes/objective_items.dm]: remove the and put name = "\proper Gooncode disk"

**Comment by Carbonhell** [comment]
code is a+ and if an head refuses i'll personally fart 50 shades of poo on him

**Comment by Nokington** [comment]
lol :+1:

**Comment by Pyko1** [comment]
Gooncode now has its own extra large disk to hold all that code.

capture

**Comment by Carbonhell** [comment]
that yellow could benefit from some more shades,ask fury perhaps

**Comment by FuryMcFlurry** [comment]
meme pr

**Comment by Pyko1** [comment]
yeah, I tried to make fancy spark effect on it, but I suck at spriting so its just a big yellow/gold colored disk.

**Comment by Nebyab** [comment]
okay

**Comment by Pyko1** [comment]
fug, thats 2 approvals from heads, merge it, merge it, merge it, merge it, merge it, merge it, before its stale.

**Comment by Lawligagger** [comment]
yes

**Comment by Limed00d** [comment]
+5

**Comment by Zadeon** [comment]
Best update since farts were added in.

**Comment by ArcLumin** [comment]
Best

**Pull request merged**